backend/frontend/templates/frontend/auth/page_password_reset_done.html

```diff
@@ -0,0 +1,6 @@
+{% extends "frontend/auth/base.html" %} {% block content %}
+<div class="mx-auto w-1/3 p-6 bg-white rounded">
+  <h1 class="text-lg">Check your inbox</h1>
+  <p>We've sent you an email with a link to reset your password.</p>
+</div>
+{% endblock %}
```

Comment:
Regarding the copy on this page can we maybe mention that you'll get the email if you previously had an account or have an existing account too? Just so it's clearer. I tried to reset the password on a non-existant user and I was not fully satisfied with the feedback message.

I don't think it's a big issue and might only make sense in my head :sweat_smile:

Reply:
Regarding emails, the dev environment is currently set to use File Backend, and it dumps the emails sent as text files inside gitignore folder.